### DISCUSSION TITLE
git remote

### DISCUSSION THREAD

**Discussion by Mariana-x1** [neuefische/questions#686] [Web]
Hallo everyone i can't solve  this problem: 
I merged  pull request and delate after this I go in main Branch and want make git pull: 

what can I make?

this is how it looks:
/p/Capstone-Projekt main ❯ git pull                               16:06:09
There is no tracking information for the current branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>

If you wish to set tracking information for this branch you can do so with:

    git branch --set-upstream-to=origin/<branch> main

~/projects/Capstone-Projekt main ❯

**Comment by MerleMak** [comment]
Hi Mariana, i think first of all you should go to GitHub and check which one of your branches has a small `default` tag on it. It should look like this:
<img width="313" alt="Screenshot 2023-08-22 at 11 55 16" src="https://github.com/neuefische/questions/assets/98704849/366986df-d240-426a-b1eb-da7eec30cc76">

**Comment by MerleMak** [comment]
If you do have a default branch check if contains all the code you wrote. If yes, the easiest is to just clone it again. 
So go to your terminal and execute `git clone <ssh-key-of-Github-repo>`

**Reply by Mariana-x1** [reply]
Danke schön

**Comment by F-Kirchhoff** [comment]
Hey Mariana,
you need to specify from which branch your main branch is supposed to get its new information. Unfortunately the branches "main" and "origin/main" (the main branch in your github repo) are not connected automatically. You can fuse them together or "track them" by executing the following command: 
```bash
git pull --set-upstream origin main
```
